ui/src/clockface/components/dropdowns/Dropdown.tsx

```diff
@@ -174,7 +179,10 @@ class Dropdown extends Component<Props, State> {
             autoHeight={true}
             maxHeight={maxMenuHeight}
           >
-            <div className="dropdown--menu">
+            <div
+              className="dropdown--menu"
+              data-testid={`dropdown--menu ${testid}`}
```

Comment:
should this just be?

```suggestion
              data-testid={testid}
```